src/App.tsx

```diff
@@ -1,15 +1,23 @@
 import React from 'react';
-
-import { DocumentList } from './features/documents/DocumentList';
+import { BrowserRouter as Router, Routes, Route, Navigate } from 'react-router-dom';
+import { Documents, Settings, Sidebar } from 'routes';
```

Comment:
I think it's a cleaner path compared to a relative path. 

However, there seems to be a disadvantage that is indistinguishable from other installed libraries.
Because of this, the prefix `@yorkie-js-sdk` was added to the JS SDK.
https://github.com/yorkie-team/yorkie-js-sdk/blame/main/tsconfig.json#L11-L14

What did you think?

Reply:
In addition, it would be good to change all relative paths as follows:

https://github.com/yorkie-team/yorkie-house/blob/e006e3bb52c740b30d8e3159ffa43b413d65345c/src/features/documents/documentsSlice.ts#L2-L3

If you wish, you can create a follow-up PR for this.

Reply:
I think it's good to add a prefix. Then, I need to change the webpack config to set up an alias. Can I use [craco](https://github.com/gsoft-inc/craco) or [react-app-rewired](https://github.com/timarney/react-app-rewired) that can override the webpack config without ejecting? Or would it be better to eject CRA?

Reply:
Yes, we can. but I'm also concerned about the increased setup complexity.
(Until you told me, I forgot that webpack config is hidden here unlike the JS SDK).

For now, It's better to clean up messy relative paths with absolute paths.